src/components/AssetScanner/AssetScanner.tsx

```diff
@@ -296,24 +304,39 @@ export class AssetScanner extends React.Component<
   private getImage(): Promise<string> {
     const { imageSrc } = this.state;
 
-    return imageSrc ? Promise.resolve(imageSrc) : this.getImageFromCanvas();
+    return imageSrc
+      ? Promise.resolve(imageSrc)
+      : this.getImageFromCanvas(this.props.cropSize);
   }
 
   // Made async to provide better UX for component
-  private getImageFromCanvas(): Promise<string> {
+  private getImageFromCanvas(cropSize?: CropSize): Promise<string> {
     const { errorVideoAccess } = ASNotifyTypes;
 
     if (!this.video) {
       this.notification(errorVideoAccess);
 
       return Promise.resolve('');
     }
-
-    const aspectRatio = this.video.videoWidth / this.video.videoHeight;
+    const { videoHeight, videoWidth } = this.video;
+    const { clientHeight, clientWidth } = scaleDomToVideoResolution(
+      videoHeight,
+      videoWidth,
+      this.video.clientHeight,
+      this.video.clientWidth
+    );
+    const cropSizeScaled = scaleCropSizeToVideoResolution(
```

Comment:
Because the DOM and the video has different resolutions, we need to scale the provided CropSize px-sizes to video resolution

Reply:
This comment will be lost forever when you merge this PR -- should it live in the code instead?

Reply:
yep, good point, add short descr to code why you doing this.

Reply:
Comment has been added to the function declaration